frontend/src/components/modals/code/PickVariableModal.vue

```diff
@@ -1,6 +1,16 @@
 <template>
     <AlgoModal title="Zaznacz zmienne" closeButtonLabel="Gotowe">
-        <AlgoDraggable id="variable-chips" :draggableList="this.$props.sceneObject.variables" />
+        <AlgoDraggable
+            id="variable-chips"
+            :draggableList="selectedVariables"
+            :on-click-close="handlePickVariable"
+            :content="
+                (variable) => {
+                    return `${variable.name}`;
+                }
```

Comment:
Powinno to się dać zrobić w taki sposób `(variable) => '${variable.name}'` (zamiast apostrofów backticki).
Ogólnie jak po prawej stronie nie otworze się bloku {} to `(a) => b` zadziała jak `(a) => {return b;}`